uiowa/uiowa
# [Bug fix] Added fix for off-canvas alerts

Resolves https://github.com/uiowa/uiowa/issues/6979

# How to test
```
ddev blt frontend && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /
```
1. Edit the slider and remove the headline from each slider and try to save the block
2. Confirm that alert message is readable and looks like the screenshots below

<img width="472" alt="Screenshot 2023-10-09 at 9 56 32 AM" src="https://github.com/uiowa/uiowa/assets/1036433/8b19b959-7f75-4714-8716-10f81a59e1c7">
<img width="491" alt="Screenshot 2023-10-09 at 9 56 23 AM" src="https://github.com/uiowa/uiowa/assets/1036433/96be7c96-1e0d-4830-a22f-72df36f0c2c8">
